apps/dialtone-documentation/docs/about/whats-new/posts/2024-7-5.md

```diff
@@ -0,0 +1,58 @@
+---
+heading: New search in the Tokens Page and new Empty State Component
+author: Nina Repetto
+posted: '2024-7-5'
+---
+
+<BlogPost :author="$frontmatter.author" :posted="parse($frontmatter.posted, 'y-M-d', new Date())" :heading="$frontmatter.heading">
+
+**👋 Hey there,** we're excited to share some updates with you:
+
+* We redesigned our [Tokens Page](/tokens) and now you can easily search and copy any token.
+
+![New Tokens Search](/assets/images/token-search.gif)
+
+* We are rolling out a new component, the [Empty State Component](/components/empty-state.html).
+
+![Empty State example](/assets/images/empty-state.png)
+
+## Redesign of the tokens page
+
+### Now it's easier to find a token
+
+Now you can search for the token you need by name, category or value.
+Once you find the token you need, you can click on the copy button to copy it in the selected format.
+
+### Examples for every token
+
+We now added examples for all the tokens to better show what the result of using the token will be.
+
+### Sidebar with categories and subcategories
+
+You can now explore all the tokens available by category and subcategory using the new sidebar on the right.
+While you search the categories will be filtered.
+
+### Change theme, format and share the filter
+
+Change theme and format in the topbar. Available formats are: **CSS**, **Android** and **iOS**. Change the theme to see how the token would look both in light and dark mode. Use the "Share filter" button to get a URL with the search and filters applied.
+
+![Token Filters](/assets/images/tokens-filters.png)
```

Comment:
Let's also mention how you can hover composition tokens to see their values, since that might not be obvious.